python/module-naming.md

```diff
@@ -0,0 +1,12 @@
+# Python module Convention > Module Naming
+
+## snake_case
+- Should be all in lowercase letters such as requests, math
+- If contains multiple words, it should be separated by dots (_) e.g. expression_engine.py
```

Comment:
It says it's should be separated by dots but I see underscore in front of it and an example with the underscore. Which one is the correct one?